README.md

```diff
@@ -46,3 +46,9 @@ cluster, and debug your running app in seconds!
 [10]: https://cloud.google.com/code/docs/vscode/install
 [11]: https://cloud.google.com/code/docs/vscode/troubleshooting
 [12]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/app_deploy_debug.gif
+
+## Support
+
+- Join our community of users using Slack channel [cloud-code-ide](https://googlecloud-community.slack.com/messages/cloud-code-ide) for Cloud Code support and outreach.
```

Comment:
So is this going to be a single Slack channel for support for all things Cloud Code IDE related and not VS Code specific? Is the IntelliJ team bought into this?